### PR TITLE
[@mantine/dates] Showing Week Numbers in Calendar Component

### PR DESCRIPTION
**Pull request by MariaBanaszkiewicz**
implements feature requested in https://github.com/orgs/mantinedev/discussions/6200

### CODE REVIEW

File: packages/@docs/demos/src/demos/dates/DatePicker/DatePicker.demo.showWeekNumbers.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const code = `
+import { DatePicker } from '@mantine/dates';
+
+function Demo() {
+  return <DatePicker showWeekNumbers />;
+}
+`;
+
+function Demo() {
+  return <DatePicker showWeekNumbers />;
```
[the remark this answers is not in the document]
Comment (MariaBanaszkiewicz) [reply packages/@docs/demos/src/demos/dates/DatePicker/DatePicker.demo.showWeekNumbers.tsx]: done

**Comment by rtivital** [comment]
Thanks!

**Comment by Kenzo-Wada** [comment]
Nice job @MariaBanaszkiewicz , this is AWESOME!

I thought that it would be helpful if it's demo shown at the docs, WDYT? https://mantine.dev/dates/calendar/

**Comment by rtivital** [comment]
I've added demos on my side already

**Comment by Kenzo-Wada** [comment]
thx!

**Pull request merged**